lawnchair/src/app/lawnchair/ui/preferences/components/colorpreference/pickers/CustomColorPicker.kt

```diff
@@ -0,0 +1,328 @@
+
+            }
+        }
+
+        val pagerState = rememberPagerState(0)
+        val scope = rememberCoroutineScope()
+        val scrollToPage =
+            { page: Int -> scope.launch { pagerState.animateScrollToPage(page) } }
+
+        PreferenceGroup(
+            heading = stringResource(id = R.string.color_sliders),
+            modifier = Modifier.padding(top = 12.dp),
+        ) {
+
+            Column {
+
+                Row(
+                    horizontalArrangement = Arrangement.spacedBy(space = 8.dp),
+                    modifier = Modifier
+                        .padding(horizontal = 16.dp)
+                        .padding(top = 12.dp),
+                ) {
+                    Chip(
+                        label = stringResource(id = R.string.hsb),
+                        onClick = { scrollToPage(0) },
+                        currentOffset = pagerState.currentPage + pagerState.currentPageOffset,
+                        page = 0,
+                    )
+                    Chip(
+                        label = stringResource(id = R.string.rgb),
+                        onClick = { scrollToPage(1) },
+                        currentOffset = pagerState.currentPage + pagerState.currentPageOffset,
+                        page = 1,
+                    )
+                }
+
+                HorizontalPager(
+                    modifier = Modifier.animateContentSize(),
+                    count = 2,
+                    state = pagerState,
+                    verticalAlignment = Alignment.Top,
+                ) { page ->
+                    when (page) {
+                        0 -> {
+                            HsvColorPicker(
+                                selectedColor = selectedColor,
+                                onSelectedColorChange = {
+                                    textFieldValue.value =
+                                        textFieldValue.value.copy(
+                                            text = intColorToColorString(
+                                                selectedColor
+                                            )
+                                        )
+                                },
+                                onSliderValuesChange = { newColor ->
+                                    focusManager.clearFocus()
+                                    onSelect(newColor)
+                                }
+                            )
+                        }
+                        1 -> {
+                            RgbColorPicker(
+                                selectedColor = selectedColor,
+                                onSelectedColorChange = {
+                                    textFieldValue.value =
+                                        textFieldValue.value.copy(
+                                            text = intColorToColorString(selectedColor)
+                                        )
```

Comment:
```suggestion
                                        ),
```

Reply:
I think you added this one by mistake. Ill skip this change. I cannot add a comma to this.